gen/qr.hoon

```diff
@@ -0,0 +1,359 @@
] [15 17] [16 16] [15 16] [16 15] [15 15] [16 14] [15 14] [16 13] [15 13] [16 12] [15 12] [16 11] [15 11] [16 10] [15 10] [16 9] [15 9] [16 8] [15 8] [16 7] [15 7] [16 5] [15 5] [16 4] [15 4] [16 3] [15 3] [16 2] [15 2] [16 1] [15 1] [16 0] [15 0] [14 0] [13 0] [14 1] [13 1] [14 2] [13 2] [14 3] [13 3] [14 4] [13 4] [14 5] [13 5] [14 7] [13 7] [14 8] [13 8] [14 9] [13 9] [14 10] [13 10] [14 11] [13 11] [14 12] [13 12] [14 13] [13 13] [14 14] [13 14] [14 15] [13 15] [14 16] [13 16] [14 17] [13 17] [14 18] [13 18] [14 19] [13 19] [14 20] [13 20] [14 21] [13 21] [14 22] [13 22] [14 23] [13 23] [14 24] [13 24] [14 25] [13 25] [14 26] [13 26] [14 27] [13 27] [14 28] [13 28] [14 29] [13 29] [14 30] [13 30] [14 31] [13 31] [14 32] [13 32] [14 33] [13 33] [14 34] [13 34] [14 35] [13 35] [14 36] [13 36] [12 36] [11 36] [12 35] [11 35] [12 34] [11 34] [12 33] [11 33] [12 32] [11 32] [12 31] [11 31] [12 30] [11 30] [12 29] [11 29] [12 28] [11 28] [12 27] [11 27] [12 26] [11 26] [12 25] [11 25] [12 24] [11 24] [12 23] [11 23] [12 22] [11 22] [12 21] [11 21] [12 20] [11 20] [12 19] [11 19] [12 18] [11 18] [12 17] [11 17] [12 16] [11 16] [12 15] [11 15] [12 14] [11 14] [12 13] [11 13] [12 12] [11 12] [12 11] [11 11] [12 10] [11 10] [12 9] [11 9] [12 8] [11 8] [12 7] [11 7] [12 5] [11 5] [12 4] [11 4] [12 3] [11 3] [12 2] [11 2] [12 1] [11 1] [12 0] [11 0] [10 0] [9 0] [10 1] [9 1] [10 2] [9 2] [10 3] [9 3] [10 4] [9 4] [10 5] [9 5] [10 7] [9 7] [10 8] [9 8] [10 9] [9 9] [10 10] [9 10] [10 11] [9 11] [10 12] [9 12] [10 13] [9 13] [10 14] [9 14] [10 15] [9 15] [10 16] [9 16] [10 17] [9 17] [10 18] [9 18] [10 19] [9 19] [10 20] [9 20] [10 21] [9 21] [10 22] [9 22] [10 23] [9 23] [10 24] [9 24] [10 25] [9 25] [10 26] [9 26] [10 27] [9 27] [10 28] [9 28] [10 29] [9 29] [10 30] [9 30] [10 31] [9 31] [10 32] [9 32] [10 33] [9 33] [10 34] [9 34] [10 35] [9 35] [10 36] [9 36] [8 28] [7 28] [8 27] [7 27] [8 26] [7 26] [8 25] [7 25] [8 24] [7 24] [8 23] [7 23] [8 22] [7 22] [8 21] [7 21] [8 20] [7 20] [8 19] [7 19] [8 18] [7 18] [8 17] [7 17] [8 16] [7 16] [8 15] [7 15] [8 14] [7 14] [8 13] [7 13] [8 12] [7 12] [8 11] [7 11] [8 10] [7 10] [8 9] [7 9] [5 9] [4 9] [5 10] [4 10] [5 11] [4 11] [5 12] [4 12] [5 13] [4 13] [5 14] [4 14] [5 15] [4 15] [5 16] [4 16] [5 17] [4 17] [5 18] [4 18] [5 19] [4 19] [5 20] [4 20] [5 21] [4 21] [5 22] [4 22] [5 23] [4 23] [5 24] [4 24] [5 25] [4 25] [5 26] [4 26] [5 27] [4 27] [5 28] [4 28] [3 28] [2 28] [3 27] [2 27] [3 26] [2 26] [3 25] [2 25] [3 24] [2 24] [3 23] [2 23] [3 22] [2 22] [3 21] [2 21] [3 20] [2 20] [3 19] [2 19] [3 18] [2 18] [3 17] [2 17] [3 16] [2 16] [3 15] [2 15] [3 14] [2 14] [3 13] [2 13] [3 12] [2 12] [3 11] [2 11] [3 10] [2 10] [3 9] [2 9] [1 9] [0 9] [1 10] [0 10] [1 11] [0 11] [1 12] [0 12] [1 13] [0 13] [1 14] [0 14] [1 15] [0 15] [1 16] [0 16] [1 17] [0 17] [1 18] [0 18] [1 19] [0 19] [1 20] [0 20] [1 21] [0 21] [1 22] [0 22] [1 23] [0 23] [1 24] [0 24] [1 25] [0 25] [1 26] [0 26] [1 27] [0 27] [1 28] [0 28] ~]
```

Comment:
somewhat stretches the definition of "constant", I feel like large chunks of this can be produced from a loop? At least like, `~[~[36 35 36^9] ~[34 33 9^36] ~[32 31 36^33 27^9] ~[30 29 9^27 33^36] ~[28 27 36^0] ~[26 25 0^36 ...]`

Reply:
Agreed, TODO in the future...